src/layout/bubbles.svelte

```diff
@@ -25,6 +25,11 @@
     { left: 92, size: 108, delay: 0, duration: 8, popping: false }
   ]
 
+  if (window.location.pathname === '/bubbles') {
```

Comment:
Oh. With this approach you are checking if path is bubbles on every single page. I think it's a overkill here.

Maybe just create a new route/page and disable inherited layout somehow?

Reply:
I moved all whole new and old bubbles logic into separated file - first file with bubbles stay without any changes.